connector/src/main/java/org/astraea/connector/backup/Exporter.java

```diff
@@ -139,21 +154,27 @@ public static class Task extends SinkTask {
 
     private final BlockingQueue<Record<byte[], byte[]>> recordsQueue = new LinkedBlockingQueue<>();
 
+    private final AtomicLong bufferSize = new AtomicLong();
```

Comment:
As it is hot code maybe we should consider using `LongAdder` instead of  `AtomicLong`